### PR TITLE
fix inconsistency in exercise topics

### LINKED ISSUE #329: Inconsistency in exercise topics
There are interesting topic names here: `algorithms` and  `algorithm`.
There's also `Control Flow If Else Statements` and `Control Flow Conditionals`.
And also `Control Flow loops` and `Loops`.

 In [problem-specifications/TOPICS.txt](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) there are `algorithms`, `loops`, `conditionals`. 

Is there a reason for this? I'd like to submit a PR to fix it, if it was not intentional.

### PR DESCRIPTION
**Pull request by heitorPB**
Are there other topics that need changes?

fix #329

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
       "difficulty": 3,
       "topics": [
         "control_flow_if_else_statements",
```
Comment (patricksjackson) [inline config.json]: I think this can also become "conditionals"

**Comment by patricksjackson** [comment]
Other topic fixes I noticed:
"mathematics" should be "math"
"data_structure" should be "data_structures" (plural)

You need to run `bin/fetch-configlet && bin/configlet fmt .` to reformat the config.json file before we can merge it.

Also if you have the time, adding a brief pointer to the topics file in the README under 'Adding an Exercise' would be very useful :)

Thanks!

**Comment by heitorPB** [comment]
Oh, i missed those two. Thanks for spotting them!

Oh, didn't know configlet is needed to update the topics as well. Travis is happy now :)

I updated the README as suggested. What do you think about it?

**Comment by patricksjackson** [comment]
Perfect! The configlet tool is used for several things, including (in this case) just making sure that the formatting is correct in the json file. It was just making sure that the list of Topics was in alphabetical order.

Thanks!

**Pull request merged**